src/Classes/Trajectory.m

```diff
@@ -1,74 +1,160 @@
-%% Trajectory - represents a (straight line) trajectory by two Point3D (entry and target) objects
-%
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dep. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicne
-% 2013 - 2017
-% mail@andreashusch.de
 classdef Trajectory < handle & matlab.mixin.Copyable & MetaTrajectory
```

Comment:
@adhusch , could you please have a look through the file and check if everything has been properly documented with the right descriptions?